### ISSUE TITLE
Expose 3rd-party libraries

### ISSUE THREAD

**Issue by cmin764** [robocorp/robocorp#208]
### Target components

docs

### Why & who

In order to teach ReMark about better suggestions when it comes to use 3rd-party libs replacing old legacy wrappers. Users asking questions in the chatbot will benefit from these new learnings.

### Feature or enhancement

Let's cover the following 3rd-party libraries:

- [x] 3rd-party: requests
- [x] 3rd-party: selenium
- [x] 3rd-party: pandas
- [x] #211
- [x] #212
- [x] #213
- [x] #214
- [x] #215
- [x] #216
- [x] #219
- [ ] 3rd-party: exchangelib
- [x] #247
- [x] #241
- [x] #238
- [x] #243
- [x] #221
- [ ] 3rd-party: java-access-bridge-wrapper
- [x] #245
- [ ] 3rd-party: Google APIs
- [x] #227

### Solution & alternative

Since this is an epic Issue, it is meant to hold all the sub-issues addressing individually each of the listed libs (or set of cohesive libs placed in one directory/README).

### How it works

1. Decide upon a bullet-point and hoover over it in order to click the target-like button which converts the list item into an individual Issue. (this way you know what is already taken and what is free to choose) 
<img width="885" alt="item-to-issue" src="https://github.com/robocorp/robocorp/assets/709053/26bc9a70-5f1a-4e59-b7ab-10552c75407e">

2. Once the list item got converted into an Issue, surf it and assign yourself to it, then groom it properly.
3. Ensure this Issue is linked to a PR that you will or already started working on. (each PR addresses a 3rd-party lib or set of closely-related libs)
4. Once the PR is merged, the Issue is automatically marked as closed and even the checkbox leading it gets marked. (if the library is already implemented, then just tick the checkbox without creating a new Issue out of it)

Good luck!

**Comment by cmin764** [comment]
Most important libs are already deployed by Ovidiu, we'll re-open if we decide to work on the rest.
